phenopacketlab-model/src/main/java/org/monarchinitiative/phenopacketlab/model/Concept.java

```diff
@@ -0,0 +1,18 @@
+package org.monarchinitiative.phenopacketlab.model;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface Concept {
```

Comment:
Suggest comments on interfaces to explain software.

Reply:
Thanks, I added the Javadoc.